runtime/gc_trace/TgcParallel.cpp

```diff
@@ -258,7 +258,7 @@ tgcHookLocalGcEnd(J9HookInterface** hook, uintptr_t eventNum, void* eventData, v
 	tgcExtensions->printf("          gc thrID     busy    stall   acquire   release   acquire   release   acquire     split avg split  alias to    deep      total deepest\n");
 	tgcExtensions->printf("                   (micros) (micros)  freelist  freelist  scanlist  scanlist      lock    arrays arraysize copycache   lists  deep objs    list\n");
 
-	scavengeTotalTime = extensions->scavengerStats._endTime - extensions->scavengerStats._startTime;
```

Comment:
Currently considering changing this to `env->_scavengerStats._workerScavengeEndTime`  - `env->_scavengerStats._workerScavengeStartTime` 

It would be more correct to get busyTime of thread relative to it’s total duration rather than the duration of the cycle (which includes times it wasn’t running)